test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -727,6 +727,40 @@ def forward(self, a, b, c, d):
         d = torch.randn(2, 3)
         self.run_test(MyModel(), (a, b, c, d))
 
+
+    def test_loop_output(self):
+        class PaddingLayer(torch.jit.ScriptModule):
+            def __init__(self):
+                super(PaddingLayer, self).__init__()
+
+            @torch.jit.script_method
+            def forward(self, input_t, shape_t):
+                # type: (Tensor, Tensor) -> Tensor
+                output_size = [len(shape_t), int(shape_t.max())]
+                model_input = torch.zeros(size=output_size, dtype=input_t.dtype, device=input_t.device)
+                s_start = 0
+                for s_id, s_len in enumerate(shape_t):
+                    model_input[s_id, :s_len] = input_t[s_start : s_start + s_len]
+                    s_start = s_start + s_len
+                return model_input
+
+        input_t = torch.ones(size=[10], dtype=torch.long)
+        shape_t = torch.tensor([1, 2, 3, 4], dtype=torch.long)
+        f = io.BytesIO()
+
+        p = PaddingLayer()
+        example_output = p(input_t, shape_t)
+
+        torch.onnx.export(p, (input_t, shape_t), f,
+                          export_params=True,
+                          opset_version=11,
+                          input_names=['input', 'shape_tensor'],
+                          output_names=['padded_input'],
+                          dynamic_axes={'input': {0: 'concat_sequence'},
+                                        'shape_tensor': {0: 'number_of_seq'},
+                                        'padded_input': {0: 'number_of_seq', 1: 'max_padded_sequence'}},
+                          example_outputs=[example_output])
```

Comment:
https://github.com/pytorch/pytorch/blob/566e06eb9b7edfe99fc4bd04ceac691b4bed1b37/torch/onnx/__init__.py#L182

btw the description of `example_outputs` requires updates, it is more than just tensors that are supported now. And besides that, still `[example_output]` is invalid format. I wonder how it actually pass the test, where it actually shouldn't ...

Reply:
well example_outputs can be a list of Tensors and [example_outputs] is a list of tensors so why it is an invalid format?